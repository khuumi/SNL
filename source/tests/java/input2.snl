recipe get_user to str:
	start switch:
		do show to "now accepting user input"
	next do
done
	
	do:
		var is input
		var is input+str
	return var
	done

start class:
	y is do get_user to input
#do get_user to x = input
	do show to y
done