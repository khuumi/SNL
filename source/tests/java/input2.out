now accepting user input
fd